libr/cons/grep.c

```diff
@@ -329,7 +329,7 @@ R_API int r_cons_grepbuf(char *buf, int len) {
 			}
 			R_FREE (cons->grep.json_path);
 		} else {
-			char *out = r_print_json_indent (buf, I(use_color));
+			char *out = r_print_json_indent (buf, I(use_color), ' ', 2);
```

Comment:
so change it to something like r-print_json_indent(const char *buf, bool color, const char *indent) {